node_modules/serve-index/public/style.css

```diff
@@ -0,0 +1,257 @@
+* {
```

Comment:
Should this have been checked in? I thought the node_modules directory was git ignored.

Reply:
It should not be, I don't know how this happened. I'll revert it.